docs/types/value-types.rst

```diff
@@ -188,25 +188,22 @@ Type conversions:
 Implicit conversions from ``address payable`` to ``address`` are allowed, whereas conversions from ``address`` to ``address payable``
 must be explicit via ``payable(<address>)``.
 
-:ref:`Address literals<address_literals>` can be implicitly converted to ``address payable``.
-
 Explicit conversions to and from ``address`` are allowed for integers, integer literals, ``bytes20`` and contract types with the following
 caveat:
 The result of a conversion of the form ``address(x)``
 has the type ``address payable``, if ``x`` is of integer or fixed bytes type,
-a literal or a contract with a receive or payable fallback function.
+or a contract with a receive or payable fallback function.
```

Comment:
The documentation change looks good otherwise (apart from the one other phrase I commented about), but this here made me stop and ask, if it still being ``address payable`` for integers and fixed bytes types is actually what we want...

Reply:
No we don't.